src/main/scripts/prepare-release.py

```diff
@@ -0,0 +1,165 @@
+
+import git
+import logging as log
+import os
+import re
+import subprocess
+import sys
+from argparse import ArgumentParser
+from datetime import datetime
+from ruamel.yaml import YAML
+from tempfile import mkstemp
+
+products = ["bamboo", "bamboo-agent",
+            "bitbucket", "confluence", "crowd", "jira"]
+prodbase = "src/main/charts"
+
+
+# TODO: As a first pass this currently only filters anything tagged
+# with 'CLIP-nnn'. However there are other options; see
+# https://hello.atlassian.net/wiki/spaces/DCD/pages/2108707663/DACI+Automating+the+Helm+release+process
+def changelog_filter(log_entry):
+    return re.match(r'^\*\s+CLIP-[0-9]+', log_entry) != None
```

Comment:
Maybe it's good to document in our process that to mention ticket number and use `CLIP-` as port of log as a mandatory step.

Reply:
I think merging with squash solve the problem of multiple commits in one PR, so merge commit message is smth to take care of (and they can be authored when a PR is merged). As to multiple PRs to the same repo - I don't see an issue her as long as it's a separate PR with its own title and changes.